src/opm/input/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -275,6 +275,10 @@ namespace Opm {
         return !m_inputNnc.input().empty();
     }
 
+    bool EclipseState::hasInputLGR() const {
+        return m_lgrs.size();
+    }
```

Comment:
Could you write this as `size() != 0`, please?  Ideally, there'd be an `empty()` predicate on the `OrderedMap<>` and we could write this as
```C++
return !m_lgrs.empty()
```
instead, but `size() != 0` is the next best thing.